README.md

```diff
@@ -115,6 +115,18 @@ The steps to prepare and trigger a release are as follows:
 
 From now on, [Travis][travis] will take over the release: build the bundles, publish to `npm` and update branch for the documentation website (see below).
 
+### Moving the `latest` dist-tag to a release:
+
+After testing the `next` release by upgrading an application, it's time to move the `latest` tag to make the release official. Use `npm dist-tag add @commercetools-frontend/ui-kit@<version> latest` to move the `latest` dist-tag onto that release.
+
+## About release-candidates & alpha / beta versions
+
+Having release candidates is not necesssary when using semantic releases:
+
+> "With semantic-release it’s discouraged to put information about stability into the version number (i.e. 1.0.0-beta or 2.0.0-rc1) because it’s mixing things up. The tool you can use to comunicate stability are npm’s dist-tags. The last paragraph of this section should give you some hints: https://github.com/semantic-release/semantic-release#how-does-it-work"
```

Comment:
Thanks for the reference.

We're not using semantic-release though, even though we follow the conventional changelog format, as we only release when we manually create a tag.
So I'm not sure we want to have `next` as the default and have to always manually move it to `latest`.

I would say that in case you wan to work on a `next` release, you switch the `tag` to `next` in travis so that releases are published there. Those might be the cases as now where we have e.g. breaking changes or we want to test things out.
I think the default should be to publish a normal release to `latest`.

Curious to hear the opinion of the others...

Reply:
I would prefer to work the way it is set up now. Otherwise we'd constantly be changing `.travis.yml` which seems to be way more work than to just move the latest tag along. Moving the latest tag is also more intuitive than changing `.travis.yml`.

Reply:
Yeah but what if you forget about it?

Reply:
Thanks for the feedback!
Alright, if you guys have more experience with that I'm fine then. Maybe I just need to get used to it and see how it goes.

🚀

Reply:
I think there is no right or wrong. We should try it and adjust and see. I am pretty optimistic that we actually like it. If we do not it‘s all cool. Nobody has to be right. It just has to be fun and productive to work with.